### PR TITLE
show defs from all sources on slang

### PR DESCRIPTION
**Pull request by inassjunus**
If urbandictionary and kamusslang has the requested definition, show both of the results, as requested by Gempal

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -337,3 +337,20 @@ def test_lookup_slang_when_both_urbandictionary_and_kamusslang_has_no_definition
     rv = lookup_slang('hitomi wo tojireba anata ga')
 
     assert rv == 'Gak nemu cuy'
+
+
+def test_lookup_slang_when_both_urbandictionary_and_kamusslang_has_definition(mocker):
```
Comment (fushar) [inline tests/test_utils.py]: - [ ] nit: `has` -> `have`

File: tululbot/utils.py [hunk 2]
```diff
@@ -120,7 +120,20 @@ def refresh_cache(self):
...
+def lookup_slang_sources(word):
+    urbandic_def = lookup_urbandictionary(word)
+    kamusslang_def = lookup_kamusslang(word)
+    if urbandic_def is not None and kamusslang_def is not None:
+        return (
+            'urbandictionary:\n{}'
+            '\n\n'
+            'kamusslang:\n{}'
+        ).format(urbandic_def, kamusslang_def)
+    else:
```
Comment (fushar) [inline tululbot/utils.py]: - [ ] this `else` is not necessary, but up to you

File: tululbot/utils.py [hunk 3]
```diff
@@ -120,7 +120,20 @@ def refresh_cache(self):
 
 def lookup_slang(word):
     not_found_word = 'Gak nemu cuy'
-    return lookup_urbandictionary(word) or lookup_kamusslang(word) or not_found_word
+    return lookup_slang_sources(word) or not_found_word
+
+
+def lookup_slang_sources(word):
+    urbandic_def = lookup_urbandictionary(word)
```
Comment (fushar) [inline tululbot/utils.py]: - [ ] please be consistent on how you abrreviate `urbandictionary`: `urban`, `urbandic`, (or `urbandict`)?

**Comment by fushar** [comment]
🚢

File: tests/test_utils.py (hunk 1 above)
Comment (inassjunus) [inline tests/test_utils.py]: hoiya :joy:

**Comment by inassjunus** [comment]
merge?

**Comment by fushar** [comment]
🚢

**Pull request merged**